erpnext/patches/v14_0/migrate_existing_lead_notes_as_per_the_new_format.py

```diff
@@ -0,0 +1,21 @@
+import frappe
+from frappe.utils import cstr, strip_html
+
+
+def execute():
+	for doctype in ("Lead", "Prospect"):
```

Comment:
Maybe add `Opportunity` too? There is not harm. If user had custom field, it will get migrated. If not, will continue after `frappe.db.has_column`